[RFC] deps: Update to libtermkey 0.18.

fwalch opened the pull request:
Same code we were using before, only from official release tarball
instead of our GitHub mirror.

justinmk commented:
Is the build failing because it's trying to use a dep from the deps repo, but bot-ci needs _this_ PR in order to update the deps repo?

fwalch commented:
@justinmk The problem is that it's looking for libunibilium.so, but only the `.a` is there because we remove `.so`s after all deps have been built. So we have to clear the cache to force a complete rebuild after merging this. Deleted the PR cache and restarted the build.

fwalch committed:
deps: Update to libtermkey 0.18.

Same code we were using before, only from official release tarball
instead of our GitHub mirror.

justinmk commented:
@fwalch Thank you. Merged, canceled build, deleted caches, then restarted build.
